### PR TITLE
docs(babelrc): add cross-env reference

### PR DESCRIPTION
**Pull request by kentcdodds**

### CODE REVIEW

File: docs/usage/babelrc.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ SET BABEL_ENV=production
 YOUR_COMMAND_HERE
 ```
 
+> If you want your command to work across platforms, try using [`cross-env`](https://www.npmjs.com/package/cross-env)
```
Comment (xtuc) [inline docs/usage/babelrc.md]: I would reword the "try using" with "you can use cross-env".

Comment (kentcdodds) [inline docs/usage/babelrc.md]: Updated. Thanks!

**Comment by existentialism** [comment]
Thanks @kentcdodds!
